src/analysis/retail/deathknight/unholy/modules/features/Abilities.tsx

```diff
@@ -55,7 +55,7 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.DARK_TRANSFORMATION.id,
         category: SPELL_CATEGORY.COOLDOWNS,
-        cooldown: 60,
+        cooldown: 60 - combatant.getTalentRank(talents.UNHOLY_COMMAND_TALENT) * 7.5,
```

Comment:
Should be `* 8` right?

`Effects          :
#1 (id=801793)   : Apply Aura (6) | Add Flat Modifier (107): Spell Cooldown (11)
                   Base Value: -8000 | Scaled Value: -8000 | PvP Coefficient: 1.00000 | Misc Value: 11 | Target: Self (1)
                   Affected Spells: Dark Transformation (63560)
                   Family Flags: 69
Description      : The cooldown of Dark Transformation is reduced by ${$s1/-1000} sec.`

Reply:
This was actually a shortcut as the talent is 8 and then 15 seconds reduction. Do you want me to change it to have an if else statement instead to correctly account for both ranks (not sure how valuable this is since I believe it's very rare anyone takes one out of two of these talent points).

Reply:
I would suggest a constant declared somewhere that you can reference like:
```typescript
export const UNHOLY_COMMAND_SCALING = [0, 8, 15];

// ...

cooldown: 60 - UNHOLY_COMMAND_SCALING[combatant.getTalentRank(talents.UNHOLY_COMMAND_TALENT)]
```

Reply:
Sounds good!